app/_src/gateway/production/breaking-changes/28x.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Kong Gateway 2.8.x and earlier breaking changes
```

Comment:
I don't like this page not having any information beyond the default copy. 

What if we had text that acknowledged that directly and linked to the other breaking changes pages?

Reply:
@cloudjumpercat we will want to list the breaking changes for 2.8.x because it's the LTS version. @lena-larionova has ideas about how this breaking changes feature can support the new LTS upgrade docs she's planning, and eng drafts have the source info we'd need to populate this page.

Reply:
+1 on needing to list all the 2.8 breaking changes, since it's LTS. Do we even know what they are though 🤔 

For the LTS to LTS upgrade, we'll need to think about how to display it. I'd want to merge this PR before attempting to do the 3.4 content.

Reply:
> +1 on needing to list all the 2.8 breaking changes, since it's LTS. Do we even know what they are though

@lena-larionova I struggled to parse out what was 2.8 breaking changes. There's one [small note in the release notes](https://docs.konghq.com/gateway/changelog/#breaking-changes-2) and we have [this section in the old upgrade guide](https://docs.konghq.com/gateway/2.8.x/install-and-run/upgrade-enterprise/#kong-manager-breaking-changes), but I wasn't sure if that section applied to 2.7 and 2.6 as well since it never mentioned 2.8.

Reply:
The section in the old upgrade guide was definitely not right, good call on leaving that out. I guess we might not have anything here 🤔